.github/workflows/github_release.yml

```diff
@@ -29,11 +29,12 @@ jobs:
         run: |
           sudo apt-get install -yq zip
           ./scripts/build_pipeline_release.sh -v "${SOURCE_TAG}" -o ./artefacts -w ./pipelines/batch-import-short-read-alignment-and-genotyping-vector/farm5/BatchImportShortReadAlignmentAndGenotyping.wdl
+          ( cd artefacts; mv BatchImportShortReadAlignmentAndGenotyping BatchImportShortReadAlignmentAndGenotyping-"${SOURCE_TAG}"; tar czf BatchImportShortReadAlignmentAndGenotyping-"${SOURCE_TAG}".tar.gz BatchImportShortReadAlignmentAndGenotyping-"${SOURCE_TAG}")
```

Comment:
I believe the dependencies are already zipped with default compression.  Not sure if gzipping an already zipped folder would inflate the size, but I suppose the wdls, options, and dependencies aren't that large in the first place, so not really an issue.

Reply:
I agree with you. The gzip choice was only because it is a typical archive format for download: tar.gz, rather than .tar alone.  Not a big deal either way I suppose